frontend/src/views/v-zoom.vue

```diff
@@ -383,8 +384,14 @@ export default {
       window.encodeHash();
     },
 
-    filterSelectedFileTypes(fileTypes) {
-      return fileTypes.filter((fileType) => this.selectedFileTypes.includes(fileType));
+    containsAtLeastOneSelected(fileTypes) {
+      let includesSelectedFileTypes = false;
+      fileTypes.forEach((fileType) => {
+        if (this.selectedFileTypes.includes(fileType)) {
+          includesSelectedFileTypes = true;
```

Comment:
This is just my personal preference, but I think it is more neat to simple return `true` here and return `false` on **line 394**. That way we can terminate the function earlier and declare one less variable. However, the difference is minimal, so it's really just my personal view. Let me know what you think :)

Reply:
I think that's a very reasonable point and I'll make the fix soon. Thank you for the comment!